bridge_adaptivity/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
 
   # Celery worker
   worker:
-    image: bridgeadaptivity_bridge
+    image: bridge_adaptivity_bridge
```

Comment:
why does this change be added?

Reply:
because it was a mistake which causes an error in building an image for celery worker.